src/components/TresCanvas.vue

```diff
@@ -66,42 +65,31 @@ const slots = defineSlots<{
 }>()
 
 
-let app: App
 
-const mountCustomRenderer = (context: TresContext) => {
-    app = createTres(slots)
-    app.provide('useTres', context) // TODO obsolete?
-    app.provide('extend', extend)
-    app.mount(scene.value)
-}
+const disableRender = computed(() => props.disableRender)
+
+const context = useTresContextProvider({
+    scene: scene.value,
+    canvas,
+    windowSize: computed(() => props.windowSize),
+    disableRender,
+    rendererOptions: props,
+})
+
+usePointerEventHandler({ scene: scene.value, contextParts: context })
 
-const dispose = () => {
-    scene.value.children = []
-    app.unmount()
-    app = createTres(slots)
-    app.provide('extend', extend)
-    app.mount(scene.value)
-    resume()
+const renderScene = () => {
+    const container = scene.value as unknown as TresObject
+    render(h(Fragment, null, slots && slots.default ? slots.default() : []), container)
 }
 
-const disableRender = computed(() => props.disableRender)
 
-onMounted(() => {
-    const existingCanvas = canvas as Ref<HTMLCanvasElement>
-
-    const context = useTresContextProvider({
-        scene: scene.value,
-        canvas: existingCanvas,
-        windowSize: props.windowSize,
-        disableRender,
-        rendererOptions: props,
-    })
 
-    usePointerEventHandler({ scene: scene.value, contextParts: context })
+onMounted(() => {
```

Comment:
actually, I think all the staff in `onMounted` can be moved to the outside, cuz all side effects be processed in composable `useTresContextProvider`, only with caution